diff --git a/mk/dev.mk b/mk/dev.mk
index 5a7c56ad..4a095fd3 100644
--- a/mk/dev.mk
+++ b/mk/dev.mk
@@ -71,6 +71,7 @@ dev/install/protoc: ## Bootstrap: Install Protoc (protobuf compiler)
 	@if [ ! -e $(PROTOC_PATH) ]; then \
 		echo "Installing Protoc $(PROTOC_VERSION) ..." \
 		&& set -x \
+		&& mkdir -p /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH) \
 		&& curl -Lo /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH).zip https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH).zip \
 		&& unzip /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH).zip bin/protoc -d /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH) \
 		&& mkdir -p $(CI_TOOLS_DIR) \
@@ -86,6 +87,7 @@ dev/install/protobuf-wellknown-types:: ## Bootstrap: Install Protobuf well-known
 	@if [ ! -e $(PROTOBUF_WKT_DIR) ]; then \
 		echo "Installing Protobuf well-known types $(PROTOC_VERSION) ..." \
 		&& set -x \
+		&& mkdir -p /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH) \
 		&& curl -Lo /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH).zip https://github.com/protocolbuffers/protobuf/releases/download/v$(PROTOC_VERSION)/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH).zip \
 		&& unzip /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH).zip 'include/*' -d /tmp/protoc-$(PROTOC_VERSION)-$(PROTOC_OS)-$(PROTOC_ARCH) \
 		&& mkdir -p $(PROTOBUF_WKT_DIR) \
